Denis_Doubinin/03.Git.Hosting/readme.md

```diff
@@ -0,0 +1,207 @@
+        if [ $a = update ]
+        then
+                echo "You don't have permissions to push your files in that repositiry"
+        else
+                echo "Push has started in the $a. Branch $b"
+                git push $a $b
+                if [ $? -eq 0 ]
+                then
+                        echo "Push to $a branch $b completed successfully"
+                else
+                        echo "Push to $a branch $b faild. Error: $?"
+                fi
+        fi
+        done
+done
+
+
+# Push results:
+
+Push has started
+Switched to branch 'dev'
+Push has started in the origin_bitbucket. Branch dev
+Everything up-to-date
+Push to origin_bitbucket branch dev completed successfully
+Push has started in the origin_github. Branch dev
+Everything up-to-date
+Push to origin_github branch dev completed successfully
+Push has started in the origin_gitlab. Branch dev
+client_global_hostkeys_private_confirm: server gave bad signature for RSA key 0
+Everything up-to-date
+Push to origin_gitlab branch dev completed successfully
+Switched to branch 'features/do_one'
+Push has started in the origin_bitbucket. Branch features/do_one
+Everything up-to-date
+Push to origin_bitbucket branch features/do_one completed successfully
+Push has started in the origin_github. Branch features/do_one
+Everything up-to-date
+Push to origin_github branch features/do_one completed successfully
+Push has started in the origin_gitlab. Branch features/do_one
+client_global_hostkeys_private_confirm: server gave bad signature for RSA key 0
+Everything up-to-date
+Push to origin_gitlab branch features/do_one completed successfully
+Switched to branch 'hotfix/we_gonna_die'
+Push has started in the origin_bitbucket. Branch hotfix/we_gonna_die
+Everything up-to-date
+Push to origin_bitbucket branch hotfix/we_gonna_die completed successfully
+Push has started in the origin_github. Branch hotfix/we_gonna_die
+Everything up-to-date
+Push to origin_github branch hotfix/we_gonna_die completed successfully
+Push has started in the origin_gitlab. Branch hotfix/we_gonna_die
+client_global_hostkeys_private_confirm: server gave bad signature for RSA key 0
+Everything up-to-date
+Push to origin_gitlab branch hotfix/we_gonna_die completed successfully
+Switched to branch 'main'
+Your branch is up to date with 'origin_gitlab/main'.
+Push has started in the origin_bitbucket. Branch main
+Everything up-to-date
+Push to origin_bitbucket branch main completed successfully
+Push has started in the origin_github. Branch main
+Everything up-to-date
+Push to origin_github branch main completed successfully
+Push has started in the origin_gitlab. Branch main
+client_global_hostkeys_private_confirm: server gave bad signature for RSA key 0
+Everything up-to-date
+Push to origin_gitlab branch main completed successfully
+Updated 0 paths from the index
+
+Beginin error
+
+Push has started in the origin_bitbucket. Branch dev1.txt
+error: src refspec dev1.txt does not match any
```

Comment:
You have too many error messages in output of your script, please check

Reply:
I see that, but I can't understand why, my script mast takes varaibles one by one from 'git brunches' (its show only my brunches) and then checkout one by one to all brunches and  push them, but it takes to varaible b not only brunches that I can see after entering the command 'git brunch' but also all files from this brunches.